libcontainer/user/user.go

```diff
@@ -433,9 +431,11 @@ func GetAdditionalGroups(additionalGroups []string, group io.Reader) ([]int, err
 // that opens the groupPath given and gives it as an argument to
 // GetAdditionalGroups.
 func GetAdditionalGroupsPath(additionalGroups []string, groupPath string) ([]int, error) {
-	group, err := os.Open(groupPath)
-	if err == nil {
```

Comment:
@cyphar, sorry for the late reply. If we pass `""` as a path or the rootfs does not contains file `/etc/group`, the `os.Open` will failed and return a `(*os.File)(nil)`, And we should not pass it to functions `GetAdditionalGroups`  and `ParseGroupFilter`, because `group != nil` in these functions always `true`.

For the case you mentioned, we can process it in `GetAdditionalGroupsPath`, Or we need not to keep that behaviour.